products/jenkins.md

```diff
@@ -28,13 +28,20 @@ releases:
     releaseLabel: "Regular"
     releaseDate: 2016-04-20
     eol: false
-    latest: "2.419"
-    latestReleaseDate: 2023-08-15
+    latest: "2.420"
```

Comment:
Strange, this was not picked up by our automation even if the release is contained in https://github.com/endoflife-date/release-data/blob/main/releases/jenkins.json.

Reply:
Opened #3492, will have a look later.